examples/launch/launch-sweeps/optuna-scheduler/README.md

```diff
@@ -0,0 +1,150 @@
+## Optuna sweep scheduler
+
+Note: This example assumes familiarity with launch setup and creating custom launch scheduler jobs. Information about launch setup can be found [here](https://docs.wandb.ai/guides/launch/quickstart) and in the "quickstart" example. For a simple example of a custom scheduler job, check out the "custom-scheduler" example in this repo. 
+
+### What is Optuna? 
+
+[Optuna](https://optuna.org/) is an open-source hyperparameter tuning [library](https://optuna.readthedocs.io/en/stable/) that exposes significant flexibility to sampling, pruning, and parameter space creation.  
+
+Using sweeps on launch, many of these features can be used to schedule wandb sweeps. To do so, use the `wandb/jobs/job-OptunaScheduler` job, or create your own using the `optuna_scheduler.py` file found in the `wandb/launch-jobs` repo [here](https://github.com/wandb/launch-jobs/jobs/sweep_schedulers/optuna_scheduler.py).
+
+
+### Run a basic example:
+
+Run a simple Optuna scheduler using out-of-the-box jobs. The scheduler and training jobs are image-sourced, and should be run on queues that can launch containers (but they do not require a builder).
+
+```bash
+wandb launch-sweep optuna_config_basic.yaml -q <container queue> -p <project>
+```
+
+
+### Key Optuna Features: 
```

Comment:
Rather than breaking into some key features here, I think for continuity, we should give the users the config and commands right away. Nothing should break the flow of running their first sweep on launch.